global/domain/v1/domain.proto

```diff
@@ -51,6 +51,8 @@ message ResourceEntity {
     TECHNICAL_USER = 15;
     AGENT = 16;
     DATA_ITEM = 17;
+    // can be user or group
+    SUBJECT = 18;
```

Comment:
This is not really a resource entity in itself. I would rather not have it because here we can use user or group directly.